pliers/extractors/models.py

```diff
@@ -116,41 +155,81 @@ def _to_df(self, result):
 
 class TFHubImageExtractor(TFHubExtractor):
 
-    ''' TFHub Extractor class for image models
+    ''' TFHub Extractor class for image models.
+
+    Note that some models may require specific input shapes.'
+    You can reshape inputs using filters, such as ImageResizingFilter.
+    ImageRescaleFilter.
+
     Args:
         url_or_path (str): url or path to TFHub model
-        features (optional): list of labels (for classification) 
-            or other feature names. If not specified, returns 
-            numbered features (feature_0, feature_1, ... ,feature_n)
-        keras_kwargs (dict): arguments to hub.KerasLayer call
+        input_dtype (optional): dtype of input data. Defaults to tf.float32
     '''
 
     _input_type = ImageStim
-    _log_attributes = ('url_or_path', 'features', 'keras_kwargs')
 
     def __init__(self, 
                  url_or_path, 
-                 features=None,
                  input_dtype=None,
-                 keras_kwargs=None):
+                 **kwargs):
         
         self.input_dtype = input_dtype if input_dtype else tf.float32
-        if keras_kwargs is None:
-            keras_kwargs = {}
-        self.keras_kwargs = keras_kwargs
 
-        logging.warning('Some models may require specific input shapes.'
-                        ' Incompatible shapes may raise errors'
-                        ' at extraction. If needed, you can reshape'
-                        ' your input image using ImageResizingFilter, '
-                        ' and rescale using ImageRescalingFilter')
```

Comment:
Yeah, I'm on the fence about this, because while I think it's a really possible common issue, it's also pretty annoying to get a warning every time you instantiate the object, and that's what documentation is for. 

We could catch TFHub errors and inject this warning-- but in cases where it just extracts the wrong thing I'm thinking that's just a user error (could happen w/o pliers as well).